src/main/java/de/tum/in/www1/exerciseapp/domain/DragAndDropQuestion.java

```diff
@@ -99,10 +105,71 @@ public DragAndDropQuestion removeDragItems(DragItem dragItem) {
         return this;
     }
 
-    public void setDragItems(Set<DragItem> dragItems) {
+    public void setDragItems(List<DragItem> dragItems) {
         this.dragItems = dragItems;
     }
-    // jhipster-needle-entity-add-getters-setters - Jhipster will add getters and setters here, do not remove
+
+    public List<DragAndDropMapping> getCorrectMappings() {
+        return correctMappings;
+    }
+
+    public DragAndDropQuestion correctMappings(List<DragAndDropMapping> dragAndDropMappings) {
+        this.correctMappings = dragAndDropMappings;
+        return this;
+    }
+
+    public DragAndDropQuestion addCorrectMappings(DragAndDropMapping dragAndDropMapping) {
+        this.correctMappings.add(dragAndDropMapping);
+        dragAndDropMapping.setQuestion(this);
+        return this;
+    }
+
+    public DragAndDropQuestion removeCorrectMappings(DragAndDropMapping dragAndDropMapping) {
+        this.correctMappings.remove(dragAndDropMapping);
+        dragAndDropMapping.setQuestion(null);
+        return this;
+    }
+
+    public void setCorrectMappings(List<DragAndDropMapping> dragAndDropMappings) {
+        this.correctMappings = dragAndDropMappings;
+    }
+    // jhipster-needle-entity-add-getters-setters - JHipster will add getters and setters here, do not remove
+
+    @PostLoad
```

Comment:
can you shortly explain why these annotations are necessary? I don't really get it why the background file path needs to be replaced?!